datashader/tests/test_quadmesh.py

```diff
@@ -188,7 +188,8 @@ def test_raster_quadmesh_upsamplex_and_downsampley(array_module):
     assert_eq_xr(res, out)
 
 
-@pytest.mark.parametrize('array_module', array_modules)
+# FIXME: dask.array does not work for this test
```

Comment:
I will open an issue, after this PR has been merged.

Reply:
I have made a fix in https://github.com/holoviz/datashader/pull/1338/commits/3c48b5929d834cfaeff5cb29173788d65528c821